docs/user/QUICKSTART.md

```diff
@@ -2,64 +2,65 @@
 This guide will help you get started with Envoy Gateway in a few simple steps.
 
 ## Prerequisites
-A Kubernetes cluster with support for services of type `LoadBalancer` or a [Kind][kind] cluster.
+A Kubernetes cluster.
 
 __Note:__ Envoy Gateway is tested against Kubernetes v1.24.1.
 
 ## Installation
-Install the Gateway API CRDs:
+* Install the Gateway API CRDs:
```

Comment:
Personally, I'm not a fan of `*`, e.g. bullets, in this context. It's not as clean of a read, makes the doc look a bit cluttered, causes text to not be aligned with code snips, etc.